shared/message/vision/Goal.proto

```diff
@@ -47,7 +47,7 @@ message Goal {
     message Measurement {
         MeasurementType type = 1;
         // Normal vectors point inwards towards the centre of the object
-        fvec3 position   = 2;
+        fvec3 rGCc       = 2;  // Spherical Coordinates (1/distance, phi, theta)
```

Comment:
Given that these inverse distance things are going to be showing up and confusing people a lot. Maybe we should make a new variable naming format for these since rGCc doesn't really describe what this is.

Maybe make a new letter for these and use that (`sGCc`)? and while your in editing NUbook you should add the ones about unit vectors

Reply:
`sGCc` for `camera` to `goal` in `camera` space in `spherical` coordinates?

Reply:
Something like that, just something so someone doesn't try and multiply it by a rotation matrix and wonder what happened